dev-perl/xml_sax/xml_sax-1.02.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="Simple API for XML"
+DESCRIPTION="XML::SAX is a SAX parser access API for Perl. It includes classes and APIs required \
+for implementing SAX drivers, along with a factory class for returning any SAX parser installed \
+on the user's system."
+HOMEPAGE="https://metacpan.org/pod/XML::SAX::Base"
+COPYRIGHT="Grant McLean"
+LICENSE="Artistic"
+REVISION="1"
+SOURCE_URI="https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-SAX-$portVersion.tar.gz"
+CHECKSUM_SHA256="4506c387043aa6a77b455f00f57409f3720aa7e553495ab2535263b4ed1ea12a"
+SOURCE_DIR="XML-SAX-$portVersion"
+PATCHES="xml_sax-$portVersion.patchset"
+ADDITIONAL_FILES="
+	install_xml_sax_pureperl.pl
+	uninstall_xml_sax_pureperl.pl
+	"
+
+ARCHITECTURES="any"
+
+POST_INSTALL_SCRIPTS="$relativePostInstallDir/install_xml_sax_pureperl.pl"
+PRE_UNINSTALL_SCRIPTS="$relativePreUninstallDir/uninstall_xml_sax_pureperl.pl"
```

Comment:
Should we use `GLOBAL_WRITABLE_FILES` instead to have the ParserDetails.ini file properly managed by the package?

One problem with that: currently the file is stored in the `site_perl` directory, so its path depends on the "short version" of perl (currently 5.40). The full path is `non-packaged/lib/perl5/site_perl/5.40/XML/SAX/ParserDetails.ini`.